### DISCUSSION TITLE
need help using this for reddit

### DISCUSSION THREAD

**Discussion by Scufyl** [MalloyDelacroix/DownloaderForReddit#319] [General]
is this not able to download from redgifs because it keeps saying failed to locate content

**Comment by MalloyDelacroix** [comment]
This has been discussed in issue #317.  Basically what happened is that redgifs used to not require authentication for GET requests.  For some reason they switched to a system that now does require authentication, but have not provided a way to get user tokens yet.

They seem to be in the process of majorly overhauling their API, and do not seem to care about breaking API consumers.  This has been fixed several times already, and they keep making more breaking changes.  I assume this issue will come up several more times before they are finished with their updates.  I'm trying to stay on top of these issues, but it's been difficult.
